extensions/script-libraries/drawing/notes/bugfix-21602.md

```diff
@@ -0,0 +1 @@
+# Remove duplicate handler function _svgParseReferenceValue
```

Comment:
@BerndN Could you please rename this file to `extensions/script-libraries/drawing/notes/21602.md`?

Our release notes builder expects the bugfix notes that are about extensions to be named `XYZ.md` instead of `bugfix-XYZ.md`. Sorry about that :)